Gopkg.lock

```diff
@@ -744,42 +744,6 @@
   revision = "1f47ba4663831f2a9c28a62a7de3ff8bc45078f0"
   version = "v1.30.0"
 
-[[projects]]
-  digest = "1:71b8d81402ecab6f1b0196ea30684b6f6df20569b72302a750ce50a5e9a74727"
-  name = "google.golang.org/protobuf"
```

Comment:
Do we really don't use protobuf?

Reply:
Reverted, it happened during deps.

Reply:
It's weird that CI didn't check that.

Reply:
Yes, it is. I dont even open Gopkg.lock change details, because it not failed.